test/services/helpers/services/storageProviders.js

```diff
@@ -0,0 +1,45 @@
+const storageProviderModel = require('../../../../src/services/storageProvider/model');
+
+let createdProviderIds = [];
+console.log(storageProviderModel);
+const removeManyProviders = (ids) => storageProviderModel.deleteMany({ _id: { $in: ids } }).lean().exec();
+
+const createTestProvider = (app, opt) => ({
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['opt' is defined but never used. Allowed unused args must match /app|req|res|next|options|params/u.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5222818)